lib/forest_liana/bootstraper.rb

```diff
@@ -32,8 +32,11 @@ def perform
     private
 
     def fetch_sti_models(model)
-      type_field = model.columns.find { |c| c.name == 'type' }
-      if type_field
+      inheritance_field = model.columns.find {
+        |c| c.name == model.inheritance_column || c.name == 'type'
+      }
```

Comment:
Agree for the CHANGELOG.
Because by default, Rails consider a model as a STI if it contains a field name 'type', nothing more :)